lib/govuk_ab_testing/requested_variant.rb

```diff
@@ -0,0 +1,45 @@
+module GovukAbTesting
+  class RequestedVariant
+    attr_reader :ab_test, :request
+
+    # @param ab_test_name [AbTest] Lowercase A/B test name, like `example`
+    # @param request [ApplicationController::Request] the `request` in the
+    # controller.
+    def initialize(ab_test, request)
+      @ab_test = ab_test
+      @request = request
+    end
+
+    # Get the bucket this user is in
+    #
+    # @return [String] the current variant, "A" or "B"
+    def variant_name
+      request.headers[ab_test.request_header] == "B" ? "B" : "A"
+    end
+
+    # @return [Boolean] if the user is to be served variant A
+    def variant_a?
```

Comment:
I think if there is a convention, then we could name it that way throughout the stack (including CDN).

But if it's intentionally left flexible (A and B - and maybe one day C - can be whatever is sensible for the experiment) then referring to them that way seems fine, as long as we're not making any assumptions about which is which outside the rendering application.